assets/templates/partials/label.tmpl

```diff
@@ -0,0 +1,10 @@
+<label
+  class="ons-radio__label"
```

Comment:
A label for a radio or a checkbox have different classes:
- `ons-radio__label`
- `ons-checkbox__label`